app/models/api/ComplianceLabour.scala

```diff
@@ -20,7 +20,7 @@ import play.api.libs.json.{Format, Json}
 
 case class ComplianceLabour(numOfWorkersSupplied: Option[Int],
                             intermediaryArrangement: Option[Boolean],
-                            supplyWorkers: Boolean)
+                            supplyWorkers: Option[Boolean])
```

Comment:
This gets used in the compliance block builder (audit too), the field should be wrapped in optional() to prevent null fields